src/main/groovy/org/jenkinsci/plugins/globalEventsPlugin/GlobalEventsPlugin.groovy

```diff
@@ -98,37 +94,22 @@ class GlobalEventsPlugin extends Plugin implements Describable<GlobalEventsPlugi
         private boolean onJobCompleted = true;
         private boolean onJobFinalized = true;
         private boolean onJobDeleted = true;
+        private boolean onNodeLaunchFailure = true;
+        private boolean onNodeOnline = true;
+        private boolean onNodeOffline = true;
+        private boolean onNodeTempOnline = true;
+        private boolean onNodeTempOffline = true;
+        private boolean onQueueWaiting = true;
+        private boolean onQueueBlocked = true;
+        private boolean onQueueBuildable = true;
+        private boolean onQueueLeft = true;
         private int scheduleTime = 0;
         private String classPath = null;
 
         void setDisableSynchronization(boolean disableSynchronization) {
             this.disableSynchronization = disableSynchronization
         }
 
-        boolean getOnJobDeleted() {
```

Comment:
Does it work for you? Because jelly usually can't show data if getter is not accessible.